### PR TITLE
Set theme jekyll-theme-slate

### PR DESCRIPTION
**Pull request by Ahmed-Alfaouq**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (carlos-ssh) [inline _config.yml:1]: **Your Portfolio, it's really nice but your are missing to fill your README file. Keep that professionalism and make that required changes:**
-Kindly update your README File.
Screen Shot 2021-08-23 at 8 19 44 AM

**Review by carlos-ssh — COMMENTED** [review]
Hi @Ahmed-Alfaouq, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carlos-ssh** [comment]
**Remember that your Pull Request needs to be open during your code review once that your project it's approved you can merge your branch.**

**Comment by Ahmed-Alfaouq** [comment]
Can you please make it clear for me?
Which things you want me to change?

**Comment by carlos-ssh** [comment]
Update your README file.

**Pull request merged**